tests/Helpers/HttpHelperUnitTest.php

```diff
@@ -160,6 +166,190 @@ public function test__callWithNotMockedEndpointInTestMode(): void
             $httpHelper->post('api/not/mocked');
         });
     }
+
+    /**
+     * @return void
+     */
+    public function test__call_WithTracerWithoutHeaderBundle_SuccessFlow(): void
+    {
+        config(['tracer.headersToPropagate' => ['x-tracer-id',],]);
+        TracerSingleton::setTraceValue('test');
+        try {
+            $httpHelper = new HttpHelper();
+            $response = $httpHelper->post('api/mocked', [RequestOptions::DEBUG => false,]);
+
+            $this->assertNotNull($response);
+        } finally {
+            TracerSingleton::setTraceValue('');
```

Comment:
Este finally esta aqui só para resetar o valor para o proximo teste??

Se for isso o ideal seria remover este finally e utilizar o setUp() ou o tearDown().

Reply:
mas ai ele vai executar pra cada teste dentro da classe, correto? nao é o que eu quero no caso

Reply:
corrigido